README.md

```diff
@@ -952,7 +952,7 @@ $ picotool reboot
 ### white-label
 
 This command allows for OTP white-labelling, which sets the USB configuration used by the device in BOOTSEL mode.
-This can be configured from a JSON file, an example of which is in [sample-wl.json](sample-wl.json).
+This can be configured from a JSON file, an example of which is in [sample-wl.json](sample-wl.json). The schema for this JSON file is [here](json/schemas/whitelabel-schema.json)
```

Comment:
It looks like this PR also moves `sample-wl.json` to `json/sample-wl.json` ? (and same comment for the other "sample" JSON files too)